kdaglas/theStore-react
# #164530011 Enable users to login to the app.

#### What does this Pr do?
Enable users to login to the app.

#### Description of the task to be implemented?
- add the login components
- add the auth reducer,
- create the login view
- add a font to be used

#### How can this be manually tested?
- Fetch and check out this branch by running `git fetch origin ft-login-164530011 && git checkout ft-login-164530011`
- Install project dependencies and start the app by running `yarn install && yarn start`

#### Related Pivotal Tracker Story
[#164530011](https://www.pivotaltracker.com/story/show/164530011)
